charts/fluent-operator/values.yaml

```diff
@@ -147,6 +147,9 @@ fluentbit:
       readFromHead: false
     systemd:
       enable: true
+      systemdFilter:
+        enable: true
```

Comment:
Do we need an enable flag here?

Reply:
@adiforluls as I've commented, we do not need a enable flag here:
![image](https://github.com/fluent/fluent-operator/assets/18525465/b287a585-f6b9-4935-ae8c-0d9f2ea0c22b)